### PR TITLE
Section 8 update

### PR DESCRIPTION
**Pull request by Pelayo-Reguera**
Update of the crosscutting concepts of the documentation

### CODE REVIEW

**Review by UO283069 — COMMENTED** [review]
In the new Domain Model there are some <<use>> that seem to be misplaced, there are also some "comment blocks" that are in Spanish, I don't really get why you are doing this and it may need to be discussed by the team if we are to leave it that way. And to conclude, you have also re-formatted some code from different classes, I don't know if something has been changed, as some imports have been removed, so please check that this does not interfere with the applications functionality and that everything works as intended.

**Comment by Pelayo-Reguera** [comment]
> In the new Domain Model there are some <<"use">> that seem to be misplaced, there are also some "comment blocks" that are in Spanish, I don't really get why you are doing this and it may need to be discussed by the team if we are to leave it that way. And to conclude, you have also re-formatted some code from different classes, I don't know if something has been changed, as some imports have been removed, so please check that this does not interfere with the applications functionality and that everything works as intended.

Obviously I will NOT leave it as it is. That is why is a DRAFT pull request and not a normal one. The imports deleted are colored in gray when they are not used in the code so it shouldn't affect the application. And the comments are there to show that I didn't find  or whatever and I want help or your opinion about that.

**Review by uo282892 — COMMENTED** [review]
About the questions in the image:
* The api is supposed to be used by the map (to retrieve places).
* Category is a field of the places.
* PlaceType is not a class, it has to be either a type or an interface extending Document to work with mongoose (MongoDB).

After solving that, it would be nice, if possible, to change a bit the background color of the folders to see them better (the lines of the folders mix with those of the classes and associations and it is a bit difficult to distinguish everything.

Finally, although I understand this is not needed for now (as it would take some time), I think it is necessary to rearrange everything to minimize the number of crossed lines and position the items as tidy as possible.

Once you solve the initial remarks, it would be ok for me in this delivery.

**Comment by Pelayo-Reguera** [comment]
@Davidmc07 which was the purpose of having a folder called place in the user POD? Are there the public places or what?

File: restapi/package-lock.json [hunk 1]
```diff
@@ -1903,9 +1903,9 @@
       "integrity": "sha512-QADzlaHc8icV8I7vbaJXJwod9HWYp8uCqf1xa4OfNu1T7JVxQIrUgOWtHdNDtPiywmFbiS12VjotIXLrKM3orQ=="
     },
     "node_modules/cookiejar": {
-      "version": "2.1.3",
-      "resolved": "https://registry.npmjs.org/cookiejar/-/cookiejar-2.1.3.tgz",
-      "integrity": "sha512-JxbCBUdrfr6AQjOXrxoTvAMJO4HBTUIlBzslcJPAz+/KT8yk53fXun51u+RenNYvad/+Vc2DIz5o9UxlCDymFQ==",
+      "version": "2.1.4",
```
Comment (UO283069) [inline restapi/package-lock.json:1906]: Can you please explain this change?

**Review by Davidmc07 — APPROVED** [review]

File: webapp/src/adapters/solid/FriendManager.ts [hunk 2]
```diff
@@ -48,12 +48,4 @@ export default class FriendManager {
         let name = getStringNoLocale(<Thing>friends, FOAF.name);
         return new User(name, webID);
     }
-
-    // public addFriend(user: User): boolean {
```
Comment (UO283069) [inline webapp/src/adapters/solid/FriendManager.ts:52]: Just a small reminder that, if this functionality is not going to be integrated, the UI must be changed 😄

File: webapp/src/adapters/solid/SolidSessionManager.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {getDefaultSession, ISessionInfo, Session} from "@inrupt/solid-client-authn-browser";
+import {getDefaultSession, Session} from "@inrupt/solid-client-authn-browser";
```
Comment (UO283069) [inline webapp/src/adapters/solid/SolidSessionManager.tsx:1]: Can you also explain this change? Thank you

Comment (Pelayo-Reguera) [reply webapp/src/adapters/solid/SolidSessionManager.tsx:1]: This import is no longer used

**Review by uo282892 — APPROVED** [review]

**Review by UO283428 — APPROVED** [review]

File: restapi/package-lock.json (hunk 1 above)
Comment (Pelayo-Reguera) [reply restapi/package-lock.json]: I supposed that changed due to "run npm install" command

**Pull request merged**